NII-DG/gogs
# Automatically insert values into affiliation master table

# PULL REQUEST

## Background

* Insert values into the affiliation master when building GIN

## Main Points of Modification

* Automatically insert values into affiliation master table

## Test

* check building successfully
* check that the table contains the values from the csv file

## Viewpoint for Review

* None

## Supplementary Information

* None

## ToDo

* None
